js/coinflex.js

```diff
@@ -0,0 +1,1885 @@
+'use strict';
+
+// ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError, BadRequest, InvalidOrder, OrderNotFound, ArgumentsRequired, InsufficientFunds } = require ('./base/errors');
+const { TICK_SIZE } = require ('./base/functions/number');
+const Precise = require ('./base/Precise');
+
+// ---------------------------------------------------------------------------
+
+module.exports = class coinflex extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'coinflex',
+            'name': 'CoinFLEX',
+            'countries': [ 'SC' ], // Seychelles
+            'rateLimit': 120, // 2500 requests per 5 minutes, 100 requests per minute
+            'version': 'v3',
+            'certified': false,
+            'userAgent': this.userAgents['chrome100'],
+            'has': {
+                'CORS': undefined,
+                'spot': true,
+                'margin': undefined,
+                'swap': true,
+                'future': true,
+                'option': undefined,
+                'addMargin': undefined,
+                'cancelAllOrders': undefined,
+                'cancelOrder': undefined,
+                'cancelOrders': undefined,
+                'createDepositAddress': undefined,
+                'createLimitOrder': true,
+                'createMarketOrder': true,
+                'createOrder': true,
```

Comment:
@ttodua Since we are already using the v2 version in some methods (fetchTrades, fetchOrders,...) can't we use it to implement methods like createOrder, cancelOrder, ...?
https://docs.coinflex.com/#rest-api-methods-private-delete-v2-cancel-orders
https://docs.coinflex.com/#rest-api-methods-private-post-v2-orders-place

Reply:
nevermind, I just saw your message to them so I'm assuming it is under your radar

Reply:
Sure, the PR is not complete, I'm working day by day atm, added new methods 👍
once all methods are unified, i'll ping you about them.